### PR TITLE
CM-148 Consolidate license directory and add Iridium license files

### PR DESCRIPTION
**Pull request by kimtrang**

### CODE REVIEW

**Review by adamcfraser — APPROVED** [review]
LGTM - does this need something to be pushed to couchbase/product-texts before it's merged?

**Comment by kimtrang** [comment]
> LGTM - does this need something to be pushed to couchbase/product-texts before it's merged?

Yes, I'll merge it when couchbase/product-texts ticket merged.  Thanks!

**Pull request merged**
